plugins/bcc-login/bcc-login.php

```diff
@@ -133,6 +133,7 @@ function get_logout_url() {
     static function activate_plugin() {
         if ( ! get_role( 'bcc-login-member' ) ) {
             add_role( 'bcc-login-member', __( 'Member' ), array( 'read' => true ) );
+            add_role( 'bcc-login-youth-member', __( 'Youth Member' ), array( 'read' => true ) );
```

Comment:
I guess you only want to add this role if this specific role does not exists, no if the bcc-login-member role does not exist, right?